web/src/client/manager.js

```diff
@@ -21,11 +21,11 @@
 
 // @ts-check
 
-import { DBusClient } from "./dbus";
+import DBusClient from "./dbus";
 import { WithStatus, WithProgress } from "./mixins";
 import cockpit from "../lib/cockpit";
 
-const MANAGER_SERVICE = "org.opensuse.DInstaller.Manager";
+const MANAGER_SERVICE = "org.opensuse.DInstaller";
```

Comment:
hmm, now I am curious why previously it works when service const was wrong?

Reply:
That service name was used as a fallback:

https://github.com/yast/d-installer/blob/8a94c2c940957d56f8b0f3eb32962bcaafaf3deb/web/src/client/manager.js#L41-L43

However, we were injecting the correct D-Bus client:

https://github.com/yast/d-installer/blob/8a94c2c940957d56f8b0f3eb32962bcaafaf3deb/web/src/client/index.js#L54-L67

Hence the `MANAGER_SERVICE` constant was not used at all.